Formula/polynote.rb

```diff
@@ -0,0 +1,52 @@
+class Polynote < Formula
+  desc "The polyglot notebook with first-class Scala support"
+  homepage "https://polynote.org/"
+  url "https://github.com/polynote/polynote/releases/download/0.2.17/polynote-dist-2.12.tar.gz"
+  version "0.2.17"
+  sha256 "76ee13b2bc8ac15173cdf0d59ebd42a254f1c2b4ba2f9db0439bccd883ebd73a"
+
+  bottle :unneeded
+
+  depends_on :java => "1.8+"
+
+  def install
+    libexec.install Dir["*"]
+
+    (bin/"polynote").write <<~EOS
+      #!/usr/bin/env bash
+      (cd #{libexec} ; ./polynote "$@")
+    EOS
+  end
+
+  test do
+    mkdir testpath/"notebooks"
+
+    assert_predicate bin/"polynote", :exist?
+    assert_predicate bin/"polynote", :executable?
+
+    output = shell_output("#{bin}/polynote version 2>&1", 1)
+    assert_match "Unknown command version", output
+
+    (testpath/"config.yml").write <<~EOS
+      listen:
+        host: 127.0.0.1
+        port: 8080
+      storage:
+        dir: #{testpath}/notebooks
+    EOS
+
+    begin
+      pid = fork do
+        exec bin/"polynote", "--config", "#{testpath}/config.yml"
+      end
+      sleep 5
+
+      assert_match "<title>Polynote</title>", shell_output("curl -s 127.0.0.1:8080")
+    ensure
+      Process.kill("TERM", pid)
+      daemon_pid = shell_output("ps -eo pid,args | grep 'polynote\.Main' | cut -d ' ' -f 1").to_i
```

Comment:
Wait, you're forking a process that forks itself now. You need to get rid of the fork block above.

Reply:
Fixed.